ci/jenkins/jobs/projects.yaml

```diff
@@ -377,3 +377,162 @@
+          wrappers:
+            - credentials-binding:
+                - text:
+                    credential-id: CODECOV_TOKEN # Jenkins secret that stores codecov token
+                    variable: CODECOV_TOKEN
+            - timeout:
+                fail: true
+                timeout: 120
+                type: absolute
+            - credentials-binding:
+                - text:
+                    credential-id: VMC_NETWORK_0
+                    variable: VMC_NETWORK_0
+                - text:
+                    credential-id: VMC_NETWORK_1
+                    variable: VMC_NETWORK_1
+                - text:
+                    credential-id: VCENTERNAME
+                    variable: VCENTERNAME
+                - text:
+                    credential-id: DATACENTERNAME
+                    variable: DATACENTERNAME
+                - text:
+                    credential-id: RESOURCEPOOLPATH
+                    variable: RESOURCEPOOLPATH
+                - text:
+                    credential-id: VC_USER
+                    variable: VC_USER
+                - text:
+                    credential-id: GOVC_PASSWORD
+                    variable: GOVC_PASSWORD
+                - text:
+                    credential-id: VC_HOST
+                    variable: VC_HOST
+                - text:
+                    credential-id: DATA_STORE
+                    variable: DATA_STORE
+                - text:
+                    credential-id: VC_CLUSTER
+                    variable: VC_CLUSTER
+                - text:
+                    credential-id: VIRTUAL_MACHINE
+                    variable: VIRTUAL_MACHINE
+                - text:
+                    credential-id: ANTREA_DOCKER_USERNAME
+                    variable: ANTREA_DOCKER_USERNAME
+                - text:
+                    credential-id: ANTREA_DOCKER_PWD
+                    variable: ANTREA_DOCKER_PWD
+                - text:
+                    credential-id: AWS_ACCESS_KEY_ID
+                    variable: AWS_ACCESS_KEY_ID
+                - text:
+                    credential-id: AWS_ACCESS_KEY_SECRET
+                    variable: AWS_ACCESS_KEY_SECRET
+                - text:
+                    credential-id: AWS_KEY_PAIR_NAME
+                    variable: AWS_KEY_PAIR_NAME
+          publishers:
+            - archive:
+                allow-empty: true
+                artifacts: "logs.tar.gz"
+                case-sensitive: true
+                default-excludes: true
+                fingerprint: false
+                only-if-success: false
+      - '{test_name}-daily':
+          test_name: nephe-test-azure
+          node: 'nephe'
+          description: 'This is the daily {test_name}.'
+          branches:
+            - '${{NEPHE_GIT_REVISION}}'
+          repo_url: '${{NEPHE_REPO}}'
+          builders:
+            - deploy-vm-and-run-azure-tests-for-nephe:
+                conformance_type: 'networkpolicy'
```

Comment:
btw, we can change 
```
- deploy-vm-and-run-azure-tests-for-nephe:
       conformance_type: 'networkpolicy'
```

to 
`- deploy-vm-and-run-azure-tests-for-nephe`

Reply:
Done